src/nvim/ex_getln.c

```diff
@@ -1911,7 +1911,8 @@ static int command_line_changed(CommandLineState *s)
     //       - Immediately undo the effects.
     State |= CMDPREVIEW;
     emsg_silent++;  // Block error reporting as the command may be incomplete
-    do_cmdline(ccline.cmdbuff, NULL, NULL, DOCMD_KEEPLINE|DOCMD_NOWAIT);
+    do_cmdline(ccline.cmdbuff, NULL, NULL,
+               DOCMD_KEEPLINE|DOCMD_NOWAIT|DOCMD_PREVIEW);
```

Comment:
`State |= CMDPREVIEW;` is set just above here, can that be checked instead of adding another flag?

Reply:
Yes, Can be used, I didn't see if the State was a global variable, and declared DOCMD_PREVIEW as flags were passed by that way, ok I will use the `State`, as it was in the use

Reply:
Done